app/models/force/application.rb

```diff
@@ -58,6 +64,11 @@ class Application < Force::ObjectBase
       { custom_api: 'hasCompletedHomebuyerEducation', domain: 'has_completed_homebuyer_education', salesforce: 'Has_Completed_Homebuyer_Education' },
       { custom_api: 'hasLoanPreapproval', domain: 'has_loan_preapproval', salesforce: 'Has_Loan_Pre_approval' },
       { custom_api: 'lendingAgent', domain: 'lending_agent', salesforce: 'Lending_Agent' },
+      { custom_api: 'shortFormPreferences', domain: 'preferences', salesforce: '?', object: Force::Preference},
+      { custom_api: 'primaryApplicant', domain: 'applicant', salesforce: '?', object: Force::ApplicationMember},
+      { custom_api: 'alternateContact', domain: 'alternate_contact', salesforce: '?', object: Force::ApplicationMember},
+      { custom_api: 'householdMembers', domain: 'household_members', salesforce: '?', object: Force::ApplicationMember},
+      { custom_api: '', domain: 'demographics', salesforce: '?', object: Force::Demographics},
```

Comment:
demographics are weird because we store them on the application object in the paper application, but we have to send them to salesforce on the primary applicant object